src/graphql/batches.js

```diff
@@ -19,6 +19,7 @@ query GET_ALL_BATCHES ($id: Int, $limit: Int, $start: Int, $sort: String, $state
         end_date
         status
         medha_area
+        mode_of_payment
         state
         enrollment_type
```

Comment:
I think all these batch fields can go into a single variable like we're doing in other files, so that we don't duplicate things here.

Let's take reference from `src/graphql/students.js` and have these fields defined at one place.

Reply:
done